sagemaker-clarify/time_series_deepar/time_series_deepar.ipynb

```diff
@@ -0,0 +1,2293 @@
+   "source": [
+    "---\n",
+    "\n",
+    "This notebook's CI test result for us-west-2 is as follows. CI test results in other regions can be found at the end of the notebook. \n",
+    "\n",
+    "![This us-west-2 badge failed to load. Check your device's internet connectivity, otherwise the service is currently unavailable](https://prod.us-west-2.tcx-beacon.docs.aws.dev/sagemaker-nb/us-west-2/sagemaker-clarify|time_series_deepar|time_series_deepar.ipynb)\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ee22a23-5ea7-4a9f-a486-36a79bb83394",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "## Runtime\n",
+    "\n",
+    "This notebook takes approximately 30 minutes to run.\n",
+    "\n",
+    "### Contents\n",
+    "\n",
+    "1. [Summary](#Summary)\n",
+    "\n",
+    "2. [Prerequisites](#Prerequisites)    \n",
+    "\n",
+    "    a. [Install Mercury](#Install-Mercury)\n",
+    "\n",
+    "    b. [Install Development SDK](#Install-Development-SDK)\n",
+    "    \n",
+    "    c. [Loading the data: Everlane Dataset](#Loading-the-data:-Everlane-Dataset)\n",
+    "    \n",
+    "    d. [Train TimeSeries Model](#Train-TimeSeries-Model)\n",
+    "    \n",
+    "    e. [Create an Endpoint from Training Job](#Create-an-Endpoint-from-Training-Job)\n",
+    "\n",
+    "4. [Time Series Explainability](#Time-Series-Explainability)\n",
+    "\n",
+    "    a. [Create `TimeSeriesDataConfig`](#Create-TimeSeriesDataConfig)\n",
+    "     \n",
+    "    b. [Create `TimeSeriesModelConfig`](#Create-TimeSeriesModelConfig)\n",
+    "     \n",
+    "    c. [Create `AsymmetricShapleyValueConfig`](#Create-AsymmetricShapleyValueConfig)\n",
+    "    \n",
+    "    d. [Create `DataConfig`](#Create-DataConfig)\n",
+    "    \n",
+    "    e. [Create `ModelConfig`](#Create-ModelConfig)\n",
+    "     \n",
+    "    f. [Setup Processor](#Setup-Processor)\n",
+    "    \n",
+    "    g. [Run Explainability Call](#Run-Explainability-Call)\n",
+    "\n",
+    "5. [Analysis Config](#Analysis-Config)\n",
+    "\n",
+    "    a. [Retrieve Config From s3](#Retrieve-Config-From-s3)\n",
+    "        \n",
+    "    b. [Display Config](#Display-Config)\n",
+    "\n",
+    "6. [Explainability Results](#Explainability-Results)\n",
+    "\n",
+    "    a. [Retrieve Results From s3](#Retrieve-Results-From-s3)\n",
+    "    \n",
+    "    b. [Display Results](#Display-Results)\n",
+    "\n",
+    "7. [Clean Up](#Clean-Up)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1985cfd8-7425-4008-bd30-0d11220f186f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Summary\n",
+    "\n",
+    "This notebook is created to demonstrate the invocation of a SageMaker Clarify explainability job for a time series DeepAR forecasting model. Given a real model and a training dataset, a processingJob will be create to analyze the SHAP score for each feature attribute."
```

Comment:
>processingJob

Should be 'processing job'.

Reply:
I change it to explainability processing job.